### PR TITLE
OPL-84: added default 'fetchCalculationRule' param based on page

### PR DESCRIPTION
**Pull request by sniedzielski**
part of TICKET: https://openimis.atlassian.net/browse/OPL-84

fixing on backend side those points:
- PaymentPlan need to show only related calcrules
- ContribtuionPlan need to show only related calcrules

### CODE REVIEW

File: src/actions.js [hunk 1]
```diff
@@ -36,6 +36,18 @@ export function fetchCalculationParamsList(className, instanceClassName, instanc
 
 export function fetchCalculationRules(params) {
     let filter = !!params !== [] ? params : null;
+    if (filter === undefined) {
+        filter = []
+    }
+    
+    var pathname = window.location.pathname 
+    if (pathname.includes('contributionPlanBundles')) {
```
Comment (delcroip) [inline src/actions.js:44]: Why didn't you use <https://github.com/openimis/openimis-be-calcrule-capitation_payment_py/blob/d86a07b349ee36fe0cbd129d2da60e696831d93d/calcrule_capitation_payment/calculation_rule.py#L30 , my apology for the not clear requirement

**Review by delcroip — CHANGES_REQUESTED** [review]

File: src/actions.js (hunk 1 above)
Comment (delcroip) [reply src/actions.js:44]: account payable for the payment_plan
account_receivable for the contribution plan
https://openimis.atlassian.net/wiki/spaces/OP/pages/3046277126/Calculation+rules#type

Comment (sniedzielski) [reply src/actions.js:44]: ok, then this could be better solution. I have to fix this. @delcroip

**Comment by sniedzielski** [comment]
@delcroip 
- https://github.com/openimis/openimis-be-calculation_py/pull/25
-https://github.com/openimis/openimis-be-calcrule_contribution_legacy_py/pull/5
- https://github.com/openimis/openimis-be-calcrule_contribution_income_percentage_py/pull/4

File: src/actions.js (hunk 1 above)
Comment (sniedzielski) [reply src/actions.js:44]: fixed!

**Comment by sniedzielski** [comment]
@delcroip I also fixed '(defect) Ensure that the calcparam are removed when changing calcrule, even if the new calcrule doens’t have any params'. You can take a look in this gif
Peek 2022-03-30 12-43

**Pull request merged**